src/apps/lwaftr/binding_table.lua

```diff
@@ -143,10 +143,31 @@ local BindingTable = {}
 function BindingTable.new(psid_map, br_addresses, softwires)
    local ret = {
       psid_map = assert(psid_map),
-      br_addresses = assert(br_addresses),
-      softwires = assert(softwires)
+      softwires = assert(softwires),
+      br_addresses = cltable.new({key_type = ffi.typeof('uint8_t[16]')})
```

Comment:
@dpino I have been going through the code trying to fix the comments searching on "br" and noticed at the top 
> The address is actually stored as an index into the BR address table, because we have space in the softwire table for a 4-byte index but not a 16-byte IPv6 value.

This PR actually stores the br address on the softwire entry as a 16-byte IPv6 address. Is this going to be a problem?